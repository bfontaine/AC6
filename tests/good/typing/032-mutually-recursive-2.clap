** int -> int -> int
def  f (x) (y) = g(x) + h(h(b))
** 'a -> 'a
with g (a) = a
** int -> int
with h (b) = b + b
