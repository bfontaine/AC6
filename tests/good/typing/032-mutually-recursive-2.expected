:- f = <code>
:- g = <code>
:- h = <code>
